src/SUMMARY.md

```diff
@@ -75,3 +75,4 @@
    - [Void Docs](./contributing/void-docs/index.md)
       - [Style Guide](./contributing/void-docs/style-guide.md)
       - [Submitting Changes](./contributing/void-docs/submitting.md)
+   - [InfraDocs](./contributing/void-docs/infradocs.md)
```

Comment:
I'm not convinced that /contributing is the best place for this.  It might be better to link from /about/meta, as the docs on infradocs aren't really about contributing.